packages/bot-web-ui/src/analytics/utils.ts

```diff
@@ -18,15 +18,16 @@ export const getRsStrategyType = (selected_strategy: string) => STRATEGIES[selec
 export const getQsActiveTabString = (tab: string) => (tab === 'TRADE_PARAMETERS' ? 'trade parameters' : 'learn more');
 
 export const getSubpageName = () => {
-    const pathname = window.location.hash;
-    if (pathname.includes('dashboard')) {
+    const active_tab = localStorage.getItem('active_tab');
```

Comment:
```suggestion
    const active_tab = localStorage.getItem('active_tab') ?? '0';
```

Can we do this?

Reply:
Putting a fallback for active_tab to '0' might result in inaccurate records. Will add a default return as 'undefined' instead. This way we can know that the tab was undefined. @shafin-deriv